fdbserver/masterserver.actor.cpp

```diff
@@ -247,6 +251,8 @@ struct MasterData : NonCopyable, ReferenceCounted<MasterData> {
 		  primaryLocality(tagLocalityInvalid),
 		  neverCreated(false),
 		  lastEpochEnd(invalidVersion),
+		  liveCommittedVersion(invalidVersion),
+	      proxyLocked(false),
```

Comment:
```suggestion
		  proxyLocked(false),
```

Reply:
@yliucode you can install clang-format as pre-commit hooks to check code style. See https://github.com/barisione/clang-format-hooks